codespell_lib/data/dictionary.txt

```diff
@@ -37781,24 +37888,56 @@ separatly->separately
 separato->separator
 separatos->separators
 separatring->separating
+separe->separate
 separed->separated
+separely->separately
+separes->separates
```

Comment:
Perhaps
```
separes->separates, spares,
```
I haven't looked if this typo exists out there, though.

Reply:
I couldn't find an example of `separe->spare`. I tried searching the next closest typo (`separingly->sparingly`), but this typo is too rare to be useful.